protos/netstate/netstate.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+package netstate;
+
+// NetState defines the interface for interacting with the network state persistence layer
+service NetState {
+  // Put formats and hands off a file path to be saved to boltdb
+  rpc Put(FilePath) returns (PutResponse);
+  // Get formats and hands off a file path to get a small value from boltdb
+  rpc Get(GetRequest) returns (GetResponse);
+  // List calls the bolt client's List function and returns all file paths
+  rpc List(ListRequest) returns (ListResponse);
+  // Delete formats and hands off a file path to delete from boltdb
+  rpc Delete(DeleteRequest) returns (DeleteResponse);
+}
+
+// FilePath is a request message for the Put rpc call
+message FilePath {
```

Comment:
honestly maybe this should just be called PutRequest to match with everything else?